packages/nextjs-template/pages/_app.tsx

```diff
@@ -45,24 +46,29 @@ function DendronApp({ Component, pageProps }: AppProps) {
   useEffect(() => {
     (async () => {
       setLogLevel("INFO");
-      logger.info({ ctx: "fetchNotes:pre" });
-      const data = await fetchNotes();
-      logger.info({ ctx: "fetchNotes:got-data" });
+      const data = await fetchSections();
       setNoteData(data);
       batch(() => {
-        dispatch(browserEngineSlice.actions.setNotes(data.notes));
+        dispatch(
+          browserEngineSlice.actions.setNotes(data.notes as NotePropsDict)
```

Comment:
this isn't actually true since `data` is `Section` instead of `NoteProps`. instead of casting it to something its not, we should add a `sections` property to `browserEngineSlice`